app/src/main/java/nerd/tuxmobil/fahrplan/congress/schedule/MainActivity.java

```diff
@@ -162,15 +166,32 @@ protected void onNewIntent(Intent intent) {
         setIntent(intent);
     }
 
-    public void onGotResponse(@NonNull FetchScheduleResult fetchScheduleResult) {
-        HttpStatus status = fetchScheduleResult.getHttpStatus();
+    @Override
+    protected void onStart() {
+        super.onStart();
+        localErrorMessaging = new LocalErrorMessaging(this, errorMessage -> {
+            showErrorDialog(errorMessage);
+            return Unit.INSTANCE;
+        });
+        localErrorMessaging.startReceiving();
+    }
+
+    @Override
+    protected void onStop() {
+        if (localErrorMessaging != null) {
+            localErrorMessaging.stopReceiving();
```

Comment:
Doesn't this mean error events are lost when they fire while the Activity is stopped, e.g. during a configuration change?